vars/shared-OpenData.var.yml

```diff
@@ -121,6 +145,17 @@ extensions:
       version: "develop"
 
 
+    CKANExtArchiver:
+      <<: *CKANExtArchiver
+     version: "develop"
```

Comment:
This branch doesn't exist yet.

Reply:
it does, https://github.com/qld-gov-au/ckanext-archiver/tree/develop